documentation/release-notes-2-8.md

```diff
@@ -174,50 +178,58 @@ ReaR 2.8 is known to not support:
 If you need to get ReaR working on an architecture that is currently not supported,
 you can buy consultancy from one of our official developers.
 
+## Unsupported Features and Known Issues
 
-# Relax-and-Recover Releases
+We are aware of the following issues and limitations in ReaR 2.8:
+
+* BTRFS support is limited to single-device configurations and doesn't cover advanced BTRFS features like spanning file systems over multiple devices, BTRFS built-in RAID and multi disk configurations. (#2028)
```

Comment:
I like this precise and exact wording very much!

At first glance I was a bit confused because
it lloked as if contradicts
https://github.com/rear/rear.github.com/pull/22#issuecomment-2539113814
```
Traditional Linux kernel software RAID0 and RAID1
is supported also with btrfs ...
```
but this is not the case because it explicitly tells
that the limitation is that one btrfs filesystem must
be on one single device and this single device
can be also a higher-level block device
e.g. of a traditional Linux kernel software RAID disk
(which consists of several lower-level physical disks).